libs/models/src/lib/state/player.ts

```diff
@@ -0,0 +1,28 @@
+import { AppRaceId } from './race';
+import { AppTournamentId } from './tournament';
+
+export enum AppPlayerState {
+  Idle = 'idle',
+  Racing = 'racing',
+}
+
+export type AppTimestamp = number;
```

Comment:
Add a jsdoc comment describing what this number represents

Reply:
Updated docs da7c49d59ee51af3ed9dcea30e7055bd99311746